.github/workflows/rust.yml

```diff
@@ -23,4 +23,5 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - run: cargo install cargo-deny
+      - run: cargo install cargo-make
```

Comment:
nit: the changes to `cargo_make.rs` should probably be in the first commit, rather than the GH workflow one.

pedantic nit: I'd also prefix this commit with "github" vs "git"